tv_apps/app-deck/js/app_deck.js

```diff
@@ -61,9 +61,27 @@
         that._spatialNavigator.focus();
         that._contextMenu = new ContextMenu();
         that._contextMenu.init(that);
+        Applications.on('install', that.onAppInstalled.bind(that));
+        Applications.on('update', that.onAppUpdated.bind(that));
+        Applications.on('uninstall', that.onAppUninstalled.bind(that));
       });
     },
 
+    _fillAppButtonIcon: function ad_fillAppButtonIcon(app, elem) {
+      Applications.getIconBlob(app.manifestURL, app.entryPoint, 180,
```

Comment:
Why do we hardcode a size here? We should create a const with window.devicePixelRatio applied. I found that our home app also had similar issue. Please update the code here and file a bug for smart-home app.